src/features/people/personPageDetailsSlice.js

```diff
@@ -0,0 +1,38 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const personPageDetails = createSlice({
+  name: "personPageDetails",
+  initialState: {
+      status: "tako",
+      person: [],
+  },
+  reducers: {
+    fetchPersonDetailsPage: () => ({
+        status: "loading",
+        person: [],
+      }),
+      fetchPersonDetailsPageSuccess: (state, { payload: details }) => ({
+        status: "loading",
+        person: [details],
+      }),
+      fetchPersonDetailsPageError: () => ({
+        status: "error",
+        person: [],
+      }),
+  },
+});
+
+export const {
+  fetchPersonDetailsPage,
+  fetchPersonDetailsPageSuccess,
+  fetchPersonDetailsPageError,
+} = personPageDetails.actions;
+
+export const selectPersonPageStates = (state) => state.personPageDetails;
```

Comment:
myślę, że można zrobić nazwę selectPersonPageState ( w liczbie pojedyńczej), a jeżeli chodzi o moje zdanie odnośnie nazewnictwa to akcje związane z pobieraniem API odnośnie bazowych informacji o osobie mozna nazwać po prostu Person czyli fetchPerson, albo fetchPersonData, a strzał po CAST i CREW możnaby nazwać fetchPersonDetails :) myślę, że może być bardziej przejrzyste, ale to tylko moje zdanie :)